### DISCUSSION TITLE
TabControl item headers go invisible after screen change

### DISCUSSION THREAD

**Discussion by gmol9knots** [AvaloniaUI/Avalonia#20606] [Q&A]
### Describe the bug

After dragging the application to a different display with different resolution some tabcontrol items go invisible and cant be seen anymore

### To Reproduce

2 differen resolution screens create a tabcontrol with multiple items (no itemssource). If you cany reproduce it multiple tabcontrols in each other

### Expected behavior

Expect the headers to stay visible

### Avalonia version

11.0.6

### OS

Windows

### Additional context

![image](https://github.com/AvaloniaUI/Avalonia/assets/108529890/dd07a741-bfec-4443-92b5-95d989407cad)
![image](https://github.com/AvaloniaUI/Avalonia/assets/108529890/b9b00ed3-2e4e-452d-964d-b4385521c233)

**Comment by timunie** [comment]
please provide a minimal sample
